dcap-ql/src/lib.rs

```diff
@@ -9,38 +9,49 @@
        html_root_url = "https://edp.fortanix.com/docs/api/")]
 
 extern crate byteorder;
+#[cfg(feature = "bindings")]
 extern crate dcap_ql_sys;
 #[macro_use]
 extern crate failure;
 #[macro_use]
 #[cfg(not(feature = "link"))]
 extern crate lazy_static;
-#[cfg(not(feature = "link"))]
+#[cfg(all(feature = "bindings", not(feature = "link")))]
 extern crate libc;
+extern crate mbedtls;
 #[macro_use]
 extern crate num_derive;
 extern crate num_traits;
+#[cfg(feature = "bindings")]
 extern crate sgx_isa;
+#[cfg(feature = "bindings")]
 extern crate sgxs_loaders;
 
+#[cfg(feature = "bindings")]
```

Comment:
Instead of repeating this attribute over and over, can you put everything related into a `mod bindings` and re-export everything?